slack/endpoint.bal

```diff
@@ -164,24 +165,24 @@ public client class Client {
     # + startOfTimeRange - Start of time range as epoch
     # + endOfTimeRange - End of time range as epoch
     # + return - Stream of MessageInfo if it is a success or an error if it is a failure
-    @display {label: "Get message history of a channel"}
-    remote isolated function getConversationHistory(@display {label: "Channel name"} string channelName, 
-                                                    @display {label: "Start of time range"} string? startOfTimeRange = (), 
-                                                    @display {label: "End of time range"}string? endOfTimeRange = ()) 
+    @display {label: "Get Messages"}
+    remote isolated function getConversationHistory(@display {label: "Channel Mame"} string channelName, 
+                                                    @display {label: "Start Of Time Range"} string? startOfTimeRange = (), 
```

Comment:
`Starting From (Timestamp)` and `Ending At  (Timestamp)` ?

Reply:
Done